packages/react/src/components/Pagination/_index.scss

```diff
@@ -31,6 +31,8 @@
 
     &__list {
       justify-content: center;
+      height: 3.5rem;
+      box-sizing: border-box;
```

Comment:
Why ? Less we enforce `height`, better I feel 😄

Reply:
Because when we remove the "Goto" component on mobile the height of the pagination changes, with gives a weird-looking result 😅